File: VB/WinSolution.Module.Win/SelectRecordInLookupViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.ExpressApp.SystemModule

Namespace AuditDemo.Module.Win
	Partial Public Class SelectRecordInLookupViewController
		Inherits ViewController(Of ListView)
		Public Sub New()
			InitializeComponent()
			RegisterActions(components)
		End Sub
		Private Sub saSelectRecordInLookup_Execute(ByVal sender As Object, ByVal e As SimpleActionExecuteEventArgs) Handles saSelectRecordInLookup.Execute
			Dim processCurrentObjectAction As SimpleAction = Frame.GetController(Of ListViewProcessCurrentObjectController)().ProcessCurrentObjectAction
			If processCurrentObjectAction.Active.ResultValue AndAlso processCurrentObjectAction.Enabled.ResultValue Then
				processCurrentObjectAction.DoExecute()
			End If
		End Sub
	End Class
End Namespace

File: CS/WinSolution.Module.Win/SelectRecordInLookupViewController.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.ExpressApp.SystemModule;

namespace AuditDemo.Module.Win {
    public partial class SelectRecordInLookupViewController : ViewController<ListView> {
        public SelectRecordInLookupViewController() {
            InitializeComponent();
            RegisterActions(components);
        }
        private void saSelectRecordInLookup_Execute(object sender, SimpleActionExecuteEventArgs e) {
            ListViewProcessCurrentObjectController listViewProcessCurrentObjectController = Frame.GetController<ListViewProcessCurrentObjectController>();
            if (listViewProcessCurrentObjectController != null) {
                SimpleAction processCurrentObjectAction = Frame.GetController<ListViewProcessCurrentObjectController>().ProcessCurrentObjectAction;
                if (processCurrentObjectAction.Active.ResultValue && processCurrentObjectAction.Enabled.ResultValue) {
                    processCurrentObjectAction.DoExecute();
                }
            }
        }
    }
}
